cmd/minikube/cmd/status.go

```diff
@@ -69,6 +88,36 @@ type Status struct {
 	Worker     bool
 }
 
+// ClusterState holds a cluster state representation
+type ClusterState struct {
```

Comment:
ideally I like to see a package called, profile (cluster) that has this in it. but I am okay for it be in CMD with the goal of keeping this PR small